proto/message.py

```diff
@@ -539,6 +577,8 @@ def __contains__(self, key):
             bool: Whether the field's value corresponds to a non-empty
                 wire serialization.
         """
+        if key in getattr(self, '_stale_fields', set()):
+            return True
```

Comment:
Aren't we guaranteed by the constructor that `_stale_fields` will exist?

Reply:
I had thought this would be the case, too, but the pattern `MyMessage.wrap(...)` uses the metaclass directly and skips the class's actual constructor, so in many cases it is (surprisingly) missing.

Reply:
FWIW, constructing a set just to get the `in` operator to return false on a missing attribute is maybe an unnecessary slowdown:  the (interned) `()` would work just as well, or you could create an empty `frozenset` at module scope for it.